### DISCUSSION TITLE
PR Best Practice: Include screenshots when changing UI components

### DISCUSSION THREAD

**Discussion by bossenti** [apache/streampipes#2234] [Dev] [answered]
Hi all,

I propose implementing a best practice for our project: including screenshots in the descriptions of pull requests (PRs) that involve UI changes. This addition aims to enhance the review process by providing visual context, making it easier for reviewers to understand the modifications. 

What are your thoughts on incorporating this idea?

**Comment by tenthe** [comment, marked as answer]
I like the idea of including screenshots
